src/elements/content-sidebar/activity-feed/task-new/Task.scss

```diff
@@ -142,3 +120,64 @@ $card-left-offset: 40px;
         margin-left: 0;
     }
 }
+
+.bcs-task-assignment-tooltip-text {
+    text-align: center;
+}
+
+.bcs-task-assignment-status {
+    display: inline-block;
+    height: 32px;
+    position: relative;
+    width: 32px;
+
+    & + .bcs-task-assignment-status,
+    & + .bcs-task-assignment-count-container {
+        margin-left: $assignment-avatar-spacing;
+    }
+}
+
+.bcs-task-assignment-status-icon {
+    @include avatar-badge-icon;
+}
+
+.bcs-task-assignment-list-flyout {
+    max-width: 100%;
+    width: 280px;
+
+    // reset base styles set by ui-elements
+    ul {
+        margin: 0;
+        padding: 0;
+    }
+}
+
+.bcs-task-assignment-list {
+    max-height: 215px;
+    overflow: hidden auto;
+    overflow-y: overlay; // nicer scroll bar for Blink engines only
+}
+
+.bcs-task-assignment-list-title {
+    font-size: 15px;
+    margin: 0 0 12px 0;
+}
+
+.bcs-task-assignment-list-item {
+    align-items: flex-start;
+    display: flex;
+    margin: 6px 0 0 0;
+    padding: 3px 0;
+}
+
+.bcs-task-assignment-list-item-avatar {
```

Comment:
These names are getting a little hairy. Would be good to do a pass to convert these to SUIT-style class names, but it can wait.